src/domupdates.js

```diff
@@ -0,0 +1,13 @@
+const domUpdates = {
```

Comment:
domUpdates looks very new I look forward to getting into this topic more

Reply:
^ agreed! definitely will need a 101 on this file